RackApp/app.rb

```diff
@@ -0,0 +1,85 @@
+require 'erb'
+require 'rack'
+require 'active_support/all'
+
+class View
+     attr_reader :person, :time
```

Comment:
Your editor is configured to use `4 spaces` as indentation, for various reason and better readability use `2 spaces `